src/test/java/com/epam/tc/hw4/ex1/FirstExercise.java

```diff
@@ -0,0 +1,80 @@
+package com.epam.tc.hw4.ex1;
+
+import com.epam.tc.hw4.BaseTest;
+import com.epam.tc.hw4.pages.IndexPage;
+import io.qameta.allure.Story;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+public class FirstExercise extends BaseTest {
+
+    @Test
+    @Story("WebElements Display")
+    public void testWebElementsDisplayWithProperText() {
```

Comment:
Please refactor your tests according to Step design pattern

Reply:
No it's not. Actually adding `io.qameta.allure.Allure.step` static methods to test methods makes the opposite effect that we expect from step-design pattern.
Let's say we have step `Assert 4 header items are displayed with proper text`
```java
        // 5. Assert that there are 4 items on the header section are displayed, and they have proper texts
        step("Assert 4 header items are displayed with proper text");
        softAssert.assertTrue(indexPage.headerHomeDisplay());
        softAssert.assertTrue(indexPage.headerContactDisplay());
        softAssert.assertTrue(indexPage.headerServiceDisplay());
        softAssert.assertTrue(indexPage.headerMetalsColorsDisplay());
        softAssert.assertEquals(indexPage.headerHomeText(), "HOME");
        softAssert.assertEquals(indexPage.headerContactText(), "CONTACT FORM");
        softAssert.assertEquals(indexPage.headerServiceText(), "SERVICE");
        softAssert.assertEquals(indexPage.headerMetalsColorsText(), "METALS & COLORS");
```
and we want to use it in multiple tests. Idea of step-design pattern is to move this code to a separate method and make it re-usable.
In your case let's keep it as it is because we missed the deadline way too much.